terraso_backend/apps/core/urls.py

```diff
@@ -16,18 +16,25 @@
 from django.urls import path
 from django.views.decorators.csrf import csrf_exempt
 
+from apps.auth.middleware import auth_optional
 from apps.core.views import (
     HealthView,
     ParseGeoFileView,
     check_restore_job_status,
     create_restore_job,
 )
 
+from .landscapes_views import LandscapeExportView
+
 app_name = "apps.core"
 
 urlpatterns = [
     path("healthz/", HealthView.as_view(), name="healthz"),
     path("admin/restore", create_restore_job),
     path("admin/restore/jobs/<int:task_id>", check_restore_job_status),
     path("gis/parse/", csrf_exempt(ParseGeoFileView.as_view()), name="parse"),
+    path(
+        "landscapes/<str:slug>/export/<str:format>",
```

Comment:
```suggestion
        "landscapes/<str:slug>/<str:format>",
```

Reply:
We don't need the word export. There's a rails pattern where all you change is the file extension. That seems simple and clean.